rules/updateRollbackFailed.js

```diff
@@ -0,0 +1,82 @@
+module.exports = {};
+module.exports.describeStacks = describeStacks;
+var AWS = require('aws-sdk');
+var queue = require('d3-queue').queue;
+var message = require('lambda-cfn').message;
+var splitOnComma = require('lambda-cfn').splitOnComma;
+var getEnv = require('lambda-cfn').getEnv;
+var util = require('util');
+var allStacks = [];
+
+module.exports.config = {
+  name: 'updateRollbackFailed',
+  sourcePath: 'rules/updateRollbackFailed.js',
+  parameters: {
+    includeResources: {
+      Type: 'String',
+      Description: 'Comma separated list of stacks which don\'t have production in their stackname'
+    }
+  },
+  statements: [
+    {
+      Effect: 'Allow',
+      Action: [
+        'cloudformation:*'
+      ],
+      Resource: '*'
+    }
+  ],
+  scheduledRule: 'rate(5 minutes)'
+};
+
+module.exports.fn = function(event, callback) {
+  var productionStacks = [];
+  var included = splitOnComma(getEnv('includeResources'));
+  var q = queue(1);
+  q.defer(describeStacks, {});
+  q.awaitAll(function (err, data) {
+    if (err) return callback(err);
+    else {
+        data[0].forEach(function(i) {
+          if(i.name.indexOf('production') > -1  && i.status.indexOf('UPDATE_COMPLETE') > -1) {
```

Comment:
@aarthykc if I am reading this correctly, this will also catch stacks that are in any of these other status states shown in the list under Stack Status Codes here http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html

Is there a reason not to check for the exact match of UPDATE_ROLLBACK_FAILED?